config/vufind/SierraRest.ini

```diff
@@ -59,6 +59,9 @@ use_prefixed_ids = false
 ; used (redirect_uri above is not set).
 username_field = "code"
 password_field = "pin"
+; Some library systems use a four digit access pin. Others prefer to have alpha numeric
+; passwords. set digits_only to true for pin functionality. Default is false (alpha-num).
+digits_only = true
```

Comment:
Just checking to see if you have any thoughts on this point, since there hasn't been further discussion on it since October. (Also happy to hear @EreMaijala's thoughts, if any).

Reply:
I think we should use the same rules as when changing a password, i.e. the [changePassword] section in ILS driver's ini:

```
; Uncomment the following lines to enable password (PIN) change
;[changePassword]
; PIN change parameters. The default limits are taken from the interface documentation.
;minLength = 4
;maxLength = 4
; See the password_pattern/password_hint settings in the [Authentication] section
; of config.ini for notes on these settings. When set here, these will override the
; config.ini defaults when Voyager is used for authentication.
;pattern = "numeric"
;hint = "Your optional custom hint can go here."

```

Reply:
(Obviously Voyager shouldn't be mentioned here)

Reply:
I'm still not sure how I've managed to miss this section of the ini so many times. I had to have looked over it hundreds of times without seeing that there was a [changePassword] section. I'll remove the custom stuff I did but I'll also have to make a change to the changePassword function in the SierraRest driver in order to remove the hard-coded digits only section of password change.